examples/Backtesting/botorch_analytical.py

```diff
@@ -124,8 +129,15 @@
     n_mc_iterations=N_MC_ITERATIONS,
 )
 
-# The following lines plot the results and save the plot in run_analytical.png
-
-sns.lineplot(data=results, x="Num_Experiments", y="Target_CumBest", hue="Scenario")
-plt.gcf().set_size_inches(24, 8)
-plt.savefig("./run_analytical.png")
+themes = json.load(open("plotting_themes.json"))
+for theme in themes:
+    sns.set_theme(style="ticks", font_scale=1.75, rc=themes[theme])
+    sns.lineplot(data=results, x="Num_Experiments", y="Target_CumBest", hue="Scenario")
+    plt.savefig(
+        Path(sys.path[0], f"botorch_analytical_{theme}.svg"),
```

Comment:
sys.path[0]?

Reply:
This is a tricky one.
We *need* the location of the file that is being executed (which is obviously not just the cwd). If we were just to execute the .py file, we could use `__file__` here without a problem. However, the example is turned into a jupyter notebook for the doc building, and a jupyter notebook does not have this field. Using `sys.path[0]` was the easiest way that I found that works reliably for both the python and the notebook version.
This might change once we fix the build directory of the documentation. Until then, I am more then happy to replace this in case you know some preferable alternative.

Reply:
@AdrianSosic : Please resolve if happy.

Reply:
Ok, if you say that's the easiest way, then let's keep it as is. I was wondering though if we should add subfolders for each example that comes with rendered images, though, since the folder gets flooded files of mixed types otherwise